docs/install.md

```diff
@@ -1,9 +1,19 @@
 # Installing the Support-Lib
 
-## conan
+## Conan
 
-!!! bug "TODO"
+The library is available at [conan-center](https://conan.io/center/djinni-support-lib):
 
-## hunter
+```text
+[requires]
+djinni-support-lib/0.0.1
+```
```

Comment:
The conan install instructions now feature a `[requires]` block. This seems to make more sense in the real world than a `conan install` command.